gobblin-restli/gobblin-job-config-service/api/src/main/pegasus/gobblin/service/JobConfig.pdsc

```diff
@@ -0,0 +1,32 @@
+{
+  "type" : "record",
+  "name" : "JobConfig",
+  "namespace": "gobblin.service",
+  "doc" : "Configuration for a job",
+  "fields" : [
```

Comment:
I think we also need a job URI. Ultimately, this is the only guaranteed unique identifier.